build_tools/tests/_therock_utils

```diff
@@ -0,0 +1 @@
+../_therock_utils/
```

Comment:
Is this symlink necessary? Could try `from ... import` or some other method of getting to the file. Symlinks are only partially supported on Windows so I'd like to avoid them if possible.

Reply:
For this kind of thing, it is better to just add a `sys.path.insert` at the top of the file to add the parent dir to the path.

Reply:
Relative import failed as the parent isn't a package which is why I added a symlink. I dropped this and use `sys.path.insert` now.